_pages/blog.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+permalink: /blog
+title: Blog
+description: Stay up to date with everything we're doing!
+---
+<main>
+	<main>
+		<h1>1418 Blog</h1>
+		<h5>Monday 1/20/20</h5>
+		<p>This is the first 1418 blog. Today's a sleepier day, but everyone is still doing good work! The programmers have been hard at work making the control panel component. Meanwhile, the mechanics have wired the primary robot and is being tested. The secondary robot has been equipped with intake/outtake and is also being tested.</p>
+		<div class="img-row">
+			<img src="/images/blog/BrandonProgramming.JPG" alt="Brandon Werbel programming.">
```

Comment:
If you use images for your blog posts you should DEFINITELY categorize them into folders by post. Otherwise it will become a huge disorganized folder and it will be much harder to keep it clean down the line.